lib/spack/spack/build_systems/cmake.py

```diff
@@ -49,6 +50,8 @@ class CMakePackage(PackageBase):
     # build-system class we are using
     build_system_class = 'CMakePackage'
 
+    spack.directives.depends_on('cmake', type='build')
```

Comment:
We definitely want thing to be readable so that other users can write their own Package subclasses or understand what is already present.